### ISSUE TITLE
Cleanup orphaned records

### ISSUE THREAD

**Issue by isoos** [dart-lang/pub-dartlang-dart#1210]
It just occurred to me that after deleting a package, we have a few records that are orphaned:
- [ ] `PackageAnalysis`, `PackageVersionAnalysis`, `Analysis`
- [ ] `Job` entries
- [ ] `History` entries (soon)
- storage on `dartdoc`'s bucket

**Comment by jakobr-google** [comment]
Yep. I've modified the remove_package tool to clean up the analysis-related records. I'll clean that up and send for review tomorrow.

I removed the Job entries and dartdoc storage manually for the two packages I've removed so far.
